openraft/tests/append_entries/t30_append_inconsistent_log.rs

```diff
@@ -83,7 +83,7 @@ async fn append_inconsistent_log() -> Result<()> {
     tracing::info!("--- restart node 1 and isolate. To let node-2 to become leader, node-1 should not vote for node-0");
     {
         router.new_raft_node_with_sto(1, sto1.clone());
-        router.isolate_node(1);
+        router.isolate_node(1, true);
```

Comment:
It's not needed to change the behavior of `isolate_node()`. 

It can be just left as is to simulate error when calling `send_xxx()`, while for this PR, use another flag to trigger the simulation of emitting error when calling `connect()`.